Dockerfile

```diff
@@ -1,22 +1,22 @@
 FROM golang:1.15-alpine AS builder
 
-ARG COMMIT_SHA
-ARG SEMVER
-ENV COMMIT_SHA=${COMMIT_SHA}
-ENV SEMVER=${SEMVER}
-
 WORKDIR /src
 RUN apk add git make
 
 # Build master of kubecfg for unreleased 32 bit changes
 RUN git clone https://github.com/bitnami/kubecfg
 RUN make -C kubecfg
 
+ARG COMMIT_SHA
```

Comment:
These changes to the Dockerfile belong to the previous commit - these were moved to better align with the new commands added to build kubecfg. And you may as well take the entrypoint change too (but technically I guess that belongs in this commit as it was to more easily allow command line flags, added in this commit).

Reply:
done.

Reply:
i kinda knew it was quite right.... i was lazy. my apologies.